splunk_connect_for_snmp/snmp/manager.py

```diff
@@ -473,8 +471,16 @@ def process_snmp_data(self, varBindTable, metrics, target, mapping={}):
                     if mapping:
                         profile = mapping.get(
                             id.replace('"', ""),
-                            mapping.get(f"{mib}::{metric}", mapping.get(mib)),
+                            mapping.get(f"{mib}::{metric}", mapping.get(mib, "")),
```

Comment:
why this option with `""` was removed? with `""` the next if `not profile` also works

Reply:
changed